0_intro/exercises/vec_strlen/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 pub fn vec_strlen(vec: Vec<&str>) -> Vec<usize> {
-    todo!()
+    vec.iter()
+       .map(|elt| { elt.len() })
```

Comment:
If it's a simple lambda, you don't need the curly braces you Java boy (but it's still completely valid)
```suggestion
       .map(|elt| elt.len())
```